index.html

```diff
@@ -0,0 +1,175 @@
+                <button type="submit" class="project">See Project</button>
+            </section>
+        </div>
+        <div class="grid-item">
+            <div class="grid-item-photo" id="photo3">
+            </div>
+            <section class="grid-item-section">
+                <h2 class="title">Tonic</h2>
+                <ul class="sub-title">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                <ul class="category">
+                    <li class="category-list"><a href="index.html">html</a></li>
+                    <li class="category-list"><a href="index.html">css</a></li>
+                    <li class="category-list"><a href="index.html">javaScript</a></li>
+                </ul>
+                <button type="submit" class="project">See Project</button>
+            </section>
+        </div>
+        <div class="grid-item">
+            <div class="grid-item-photo" id="photo4">
+            </div>
+            <section class="grid-item-section">
+                <h2 class="title">Multi-Post Stories</h2>
+                <ul class="sub-title">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                <ul class="category">
+                    <li class="category-list"><a href="index.html">html</a></li>
+                    <li class="category-list"><a href="index.html">css</a></li>
+                    <li class="category-list"><a href="index.html">javaScript</a></li>
+                </ul>
+                <button type="submit" class="project">See Project</button>
+            </section>
+        </div>
+    </div>
+    <section class="about" id="about">
+        <div class="about-content1">
+            <h2>About Myself</h2>
+            <p>
+                Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <p class="connect">LET'S CONNECT</p>
+            <ul class="social-1">
+                <li><a href="index.html"><img src="image/1.png" alt="Twitter"/></a></li>
+                <li><a href="index.html"><img src="image/2.png" alt="Linkedin"/></a></li>
+                <li><a href="index.html"><img src="image/3.png" alt="Medium"/></a></li>
+                <li><a href="index.html"><img src="image/4.png" alt="Github"/></a></li>
+                <li><a href="index.html"><img src="image/5.png" alt="Angellist"/></a></li>
```

Comment:
- [ ] Your social icons seem not to be showing. Kindly adjust the paths of the image properly to display the icons 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/213791034-8759dc06-2552-4f68-ba09-712bdb05853e.png)